### DISCUSSION TITLE
Child thread cannot get traceid

### DISCUSSION THREAD

**Discussion by xiaohei88** [apache/skywalking#9333] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

I want to get traceid in the sub thread。for example：

```java
    final ExecutorService executorService = Executors.newFixedThreadPool(2);
    
    @RequestMapping("/test1/log")
    public String get1() {
        logger.info(Thread.currentThread().getName() + "------->" + TraceContext.traceId());
        
        executorService.submit(() -> {
            logger.info(Thread.currentThread().getName() + "------->" + TraceContext.traceId());
        });
        return "1";
    }
```
result: 
http-nio-8090-exec-1------->Ignored_Trace
pool-8-thread-1------->N/A

### What you expected to happen

http-nio-8090-exec-1------->Ignored_Trace
pool-8-thread-1------->Ignored_Trace

### How to reproduce

no code

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
The simple answer is, that it should not have a trace ID. Don't submit an issue rather than a discussion randomly. 

Learn more about the tracing, and manual APIs for crossing threads propagation.
